src/back/TicketManager/src/PaymentService/TicketManager.PaymentService.Services/Endpoints/Payments/CreatePaymentEndpoint.cs

```diff
@@ -0,0 +1,31 @@
+using FastEndpoints;
+using TicketManager.PaymentService.Contracts.Payments;
+using TicketManager.PaymentService.Domain.Payments;
+using TicketManager.PaymentService.Services.DataAccess;
+
+namespace TicketManager.PaymentService.Services.Endpoints.Payments;
+
+public class CreatePaymentEndpoint : EndpointWithoutRequest<PaymentTokenResponse>
+{
+    private readonly Repository<Payment, Guid> _payments;
+
+    public CreatePaymentEndpoint(Repository<Payment, Guid> payments)
+    {
+        _payments = payments;
+    }
+
+    public override void Configure()
+    {
+        Post("/payment/create");
+        AllowAnonymous();
+    }
+
+    public override async Task HandleAsync(CancellationToken ct)
+    {
+        var payment = new Payment();
+
+        await _payments.AddAsync(payment, ct);
+
+        await SendAsync(new PaymentTokenResponse() { Token = payment.Id }, cancellation: ct);
```

Comment:
You can use `SendOkAsync`.

Reply:
Also `cancellation:` is not necessary here then

Reply:
It is if you use `SendAsync` instead of `SendOkAsync`.

Reply:
Addressed in `8a0f91ef`.